ts/defs.bzl

```diff
@@ -400,7 +431,16 @@ def ts_project(
     ts_project_rule(
         name = tsc_target_name,
         srcs = srcs,
-        args = args,
+        args = args + select(
+            {
+                "//ts:skip_lib_check.always": ["--skipLibCheck"],
```

Comment:
what happens when users set both?

Reply:
They get the same error as if they select neither.
We could easily improve that though, by having a config_setting that's just for error reporting - I'll add now

Reply:
err no, I'm wrong, if one of the config_settings matches in the select then I don't know how to print an error.

They'll just have to figure out from how the error says "exactly one must be set"

Reply:
hmm, maybe it's better to have a string flag/enum? that may allow users to use the flag repeatedly as well. (in places such as .bazelrc.user)

I am thinking `always/honor_tsconfig/never`

Reply:
I don't have a way to implement `never` - we either inject an extra command-line arg or we don't.
Does that change it for you?

Reply:
can't `--skipLibCheck` take true/false as a flag value? and when you do it overrides whatever is in tsconfig?

Reply:
right... I filed that issue HAHA https://github.com/microsoft/TypeScript/issues/10967

However, I still don't think `never` is useful, we never wished for a way to turn *on* libCheck globally overriding what's in tsconfig. And we still don't wish for that. Would it be useful somehow?

Reply:
it might be useful for some. we can leave it out in this PR and leave the possibility of adding it open with enum string flags.

Reply:
hm in my testing with `string_flag(values=["x", "y"])` I'm not getting any error if I give some other value... something seems broken there

Reply:
that's weird. maybe an old version of skylib? https://github.com/bazelbuild/bazel-skylib/blob/141432789c92e9db2402ef0be58e2a2d2c4dd1fd/rules/common_settings.bzl#L81-L98

Reply:
something on my swc branch. Tested here and it works correctly if illegal value is provided.

string_flag LGTM - PTAL